### PR TITLE
`no-unknown-animations` now ignores `ease` value.

### PR DESCRIPTION
**Pull request by evilebottnawi**
https://github.com/stylelint/stylelint/issues/1240

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 - Fixed: `declaration-block-no-ignored-properties` now detects use of `overflow`, `overflow-x` and `overflow-y` with inline elements.
 - Fixed: some rules now better handle case insensitive CSS identifiers.
 - Fixed: `font-family-name-quotes`, `media-feature-no-missing-punctuation`, `media-query-list-comma-newline-after`, `media-query-list-comma-newline-before`, `media-query-list-comma-space-after` and `media-query-list-comma-space-before` rules now better ignore SCSS, Less variables and nonstandard at-rules.
+- Fixed: `no-unknown-animations` now ignores `easy` value.
```
Comment (davidtheclark) [inline CHANGELOG.md]: `ease` value

**Comment by davidtheclark** [comment]
Just the one correction, then feel free to merge.

**Comment by evilebottnawi** [comment]
CI green, merge.

**Pull request merged**